app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/bruker/BrukerRepository.kt

```diff
@@ -330,10 +333,86 @@ class BrukerRepositoryImpl(
+                            from mottaker_altinn_rolle rol
+                            join mine_altinnrolletilganger at on 
+                                rol.virksomhet = at.virksomhetsnummer and
+                                rol.role_definition_id = at."roleDefinitionId" and
+                                rol.role_definition_code = at."roleDefinitionCode"
+                            where
+                                rol.notifikasjon_id is not null
+                        ),
+                        mine_digisyfo_notifikasjoner as (
+                            select notifikasjon_id 
+                            from mottaker_digisyfo_for_fnr
+                            where fnr_leder = ? and notifikasjon_id is not null
+                        ),
+                        mine_saksfrister as (
+                            select
+                                jsonb_build_object(
+                                    'sakId', s."sakId",
+                                    'virksomhetsnummer', s.virksomhetsnummer,
+                                    'tittel', s.tittel,
+                                    'lenke', s.lenke,
+                                    'merkelapp', s.merkelapp,
+                                    'grupperingsid', s.grupperingsid,
+                                    'statuser', s.statuser,
+                                    'sist_endret', s.sist_endret
+                                ) as sak,
+                                jsonb_build_object(
+                                    'frister', 
+                                    coalesce(
+                                        json_agg(n.frist order by n.frist) filter (
+                                            where n.tilstand = '${ProdusentModel.Oppgave.Tilstand.NY}'
+                                                and n.id in (
+                                                    select * from mine_digisyfo_notifikasjoner
+                                                        union
+                                                    select * from mine_altinn_notifikasjoner
+                                                        union
+                                                    select * from mine_altinn_reportee_notifikasjoner
+                                                        union
+                                                    select * from mine_altinn_rolle_notifikasjoner
+                                                )
+                                        ), 
+                                        '[]'::json
+                                    )
+                                ) as frister
+                            from mine_saker_paginert as s
+                            left join notifikasjon as n on n.grupperingsid = s.grupperingsid
+                            group by
+                                s."sakId", -- TODO: få til å grupper på kun sakId ?
```

Comment:
jeg mistenker at vi må leve med dette. antakeligvis bare å fjerne todo
> When GROUP BY is present, or any aggregate functions are present, it is not valid for the SELECT list expressions to refer to ungrouped columns except within aggregate functions or when the ungrouped column is functionally dependent on the grouped columns, since there would otherwise be more than one possible value to return for an ungrouped column. A functional dependency exists if the grouped columns (or a subset thereof) are the primary key of the table containing the ungrouped column.

 https://www.postgresql.org/docs/current/sql-select.html

Reply:
i lys av det so mstår i docen så testet jeg med å legge en agg rundt, og det funket.
0276cf31028318a9ce0b4146903b2408194c131f